### PR TITLE
Katie doc edits

### PR DESCRIPTION
**Pull request by kkowalsky**
Revising the documentation @nvkelso started to fit Mapzen's writing style guide. 

Still needs some love, here's the TODO:

- [x] images from @nvkelso of the terrain tiles

- [x] build-a-map.md needs to be finished

- [ ] @rmglennon needs to do writing style checks

### CODE REVIEW

**Comment by kevinkreiser** [comment]
probably didnt mean to check in that `.DS_Store` file

File: docs/use-service.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+#### Additional Amazon S3 Endpoints
+
+If you’re building in Amazon AWS we recommend using machines in the `us-east` region (the same region as the S3 bucket) and use the following endpoints for increased performance:
+
+* `https://s3.amazonaws.com/elevation-tiles-prod/terrarium/{z}/{x}/{y}.png`
+* `https://s3.amazonaws.com/elevation-tiles-prod/normal/{z}/{x}/{y}.png`
+* `https://s3.amazonaws.com/elevation-tiles-prod/geotiff/{z}/{x}/{y}.tif`
+* `https://s3.amazonaws.com/elevation-tiles-prod/skadi/{N|S}{y}/{N|S}{y}{E|W}{x}.hgt.gz`
+
+NOTE: The S3 tiles are meant for efficent networking with EC2 resources only. The Amazon S3 endpoints are not cached using Cloudfront, but you could put your own Cloudfront or other CDN in fron tof them.
```
Comment (zerebubuth) [inline docs/use-service.md]: s/fron tof/front of/

File: docs/formats.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    table.append(6000 + 100 * i)
+```
+
+To decode quantized height value:
+
+  `255 - bisect.bisect_left(HEIGHT_TABLE, h)`
+
+## Skadi
+
+**Skadi** format tiles are raw elevation data in unprojected latlng (EPSG:4326) 1°x1° tiles, used by the Mapzen Terrain Service. Essentially they are the SRTMGL1 format tiles but with global coverage. See [GDAL documentation](http://www.gdal.org/frmt_various.html#SRTMHGT)) for more information.
```
Comment (zerebubuth) [inline docs/formats.md]: Is it worth mentioning that they're compressed with `gzip`? It is implied by the `.hgt.gz` file extension, but might be worth being explicit about it.

File: docs/build-a-map.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+* **Walkabout demo:** [preview](http://tangrams.github.io/walkabout-style-more-labels) | [source code](http://github.com/tangrams/walkabout-style-more-labels)
+
+![image](https://cloud.githubusercontent.com/assets/853051/11137284/13e3a5f0-896b-11e5-9ab9-be51ecb388d8.png)
+
+## GDAL
+
+GDAL allows Desktop analytics, hill shading, and contouring.
+
+_TODO: script needs to be writen to download tiles in a given area and hillshade them, show result in QGIS_
+
```
Comment (zerebubuth) [inline docs/build-a-map.md]: Is this related to #90?

File: docs/build-a-map.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+_See:_
+
+- http://jgomezdans.github.io/stitching-together-modis-data.html
+- https://gist.github.com/jgomezdans/3152468
+- http://www.mikejcorey.com/wordpress/2011/02/05/tutorial-create-beautiful-hillshade-maps-from-digital-elevation-models-with-gdal-and-mapnik/
+- https://www.mapbox.com/tilemill/docs/guides/terrain-data/#creating-hillshades
+- http://www.gdal.org/gdal_merge.html
+
+## QGIS
```
Comment (zerebubuth) [inline docs/build-a-map.md]: Does this section need to be written, or should we leave it out for the time being?

Comment (zerebubuth) [inline docs/build-a-map.md]: Oops, I see that was already on the TODO, sorry!

**Comment by zerebubuth** [comment]
Great docs, thanks! These will be really useful.

File: docs/README.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
+# Mapzen Terrain Tile documentation
+The [Mapzen terrain tiles](https://mapzen.com/projects/joerd) provide basemap terrain coverage of the world in a raster tile format.
```
Comment (nvkelso) [inline docs/README.md]: The project url should be `https://mapzen.com/projects/terrain-tiles` (not using joerd Github project name).

File: docs/README.md [hunk 6]
```diff
@@ -0,0 +1,10 @@
...
+
+This repository contains the documentation source files for Mapzen's Terrain Tiles:
+
+## [https://mapzen.com/documentation/terrain](https://mapzen.com/documentation/terrain)
```
Comment (nvkelso) [inline docs/README.md]: The sister project is `vector-tiles`, so I think the URL for terrain would be:

- `https://mapzen.com/documentation/terrain-tiles/`

File: docs/api-keys-and-rate-limits.md [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+3. Create a new key for Mapzen Terrain Tiles, and optionally, give it a name so you can remember the purpose of the project.
+4. Copy the key into your code.
+
+## Rate limits
+
+Mapzen Terrain Tiles are a free data product that is 100% cached.
+
+If you experience slow tile loading in a map area, it's likely because you are a first requester in your region of the world. Subsequent loads of the same map area should be much faster because the tile is now available in the local edge cache.
+
+If you have questions, contact [tiles@mapzen.com](mailto:tiles@mapzen.com). You can also set up your own instance of [Joerd](https://github.com/tilezen/joerd), which has access to the same data used in the Mapzen Terrain Tiles data product.
```
Comment (nvkelso) [inline docs/api-keys-and-rate-limits.md]: data product or service?

Comment (kkowalsky) [inline docs/api-keys-and-rate-limits.md]: we should change to service

**Comment by kkowalsky** [comment]
thanks for catching that @kevinkreiser 😅

@zerebubuth IMO I think we can wait on the tutorial and not rush it.

File: docs/README.md (hunk 5 above)
Comment (kkowalsky) [inline docs/README.md]: not sure if we're going to have a project page for terrain tiles yet, so holding off on this link

File: docs/use-service.md [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+
+Mapzen raster terrain tiles can be returned in the following formats:
+
+#### Requesting tiles
+
+You can request tiles using Mapzen's global CDN:
+
+- **Terrarium**
+
+  `https://tile.mapzen.com/terrarium/v1/{z}/{x}/{y}.png?api_key=terrain-tiles-xxxxxxx`
```
Comment (nvkelso) [inline docs/use-service.md]: `https://tile.mapzen.com/mapzen/terrain/v1/normal/{z}/{x}/{y}.png?api_key=terrain-tiles-xxxxxxx`

File: docs/use-service.md [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+
+- **Normal**
+
+  `
+https://tile.mapzen.com/normal/v1/{z}/{x}/{y}.png?api_key=terrain-tiles-xxxxxxx`
```
Comment (nvkelso) [inline docs/use-service.md]: `https://tile.mapzen.com/mapzen/normal/v1/{z}/{x}/{y}.png?api_key=terrain-tiles-xxxxxxx`

File: docs/build-a-map.md (hunk 3 above)
Comment (nvkelso) [inline docs/build-a-map.md]: Yes, text here needs to reference that script and show example usage.

File: docs/api-keys-and-rate-limits.md [hunk 10]
```diff
@@ -0,0 +1,22 @@
...
+## Obtain an API key
+
+To use Mapzen Terrain Tiles, you should first obtain a free developer API key. Sign in at https://mapzen.com/developers to create and manage your API keys.
+
+1. Go to https://mapzen.com/developers.
+2. Sign in with your GitHub account. If you have not done this before, you need to agree to the terms first.
+3. Create a new key for Mapzen Terrain Tiles, and optionally, give it a name so you can remember the purpose of the project.
+4. Copy the key into your code.
+
+## Rate limits
```
Comment (nvkelso) [inline docs/api-keys-and-rate-limits.md]: We need to update https://mapzen.com/documentation/overview/#mapping-products to restate these limits?

File: docs/attribution.md [hunk 11]
```diff
@@ -0,0 +1,103 @@
+# Attribution
+
+Attribution is required for many of our data providers. Example language is provided below, but you are responsible for researching each project to follow their license terms. More details are available on the [Data Sources](data-sources.md) page.
+
+**Required attribution:**
```
Comment (nvkelso) [inline docs/attribution.md]: We also need to update https://mapzen.com/rights/

File: docs/use-service.md [hunk 12]
```diff
@@ -0,0 +1,58 @@
...
+  `
+https://tile.mapzen.com/geotiff/v1/{z}/{x}/{y}.tif?api_key=terrain-tiles-xxxxxxx`
+
+  Note: GeoTIFF format tiles are 512x512 sized so request the parent tile’s coordinate. For instance, if you’re looking for a zoom 14 tile then request the parent tile at zoom 13.
+
+
+- **Skadi**
+
+  `
+https://tile.mapzen.com/skadi/v1/{N|S}{y}/{N|S}{y}{E|W}{x}.hgt.gz?api_key=terrain-tiles-xxxxxxx`
```
Comment (nvkelso) [inline docs/use-service.md]: `https://tile.mapzen.com/mapzen/skadi/v1/{N|S}{y}/{N|S}{y}{E|W}{x}.hgt.gz?api_key=terrain-tiles-xxxxxxx`

File: docs/use-service.md [hunk 13]
```diff
@@ -0,0 +1,58 @@
...
+
+- **Normal**
+
+  `
+https://tile.mapzen.com/normal/v1/{z}/{x}/{y}.png?api_key=terrain-tiles-xxxxxxx`
+
+- **GeoTIFF**
+
+  `
+https://tile.mapzen.com/geotiff/v1/{z}/{x}/{y}.tif?api_key=terrain-tiles-xxxxxxx`
```
Comment (nvkelso) [inline docs/use-service.md]: `https://tile.mapzen.com/mapzen/geotiff/v1/{z}/{x}/{y}.tif?api_key=terrain-tiles-xxxxxxx`

File: docs/index.md [hunk 14]
```diff
@@ -0,0 +1,11 @@
+The [Mapzen terrain tiles](https://mapzen.com/projects/joerd) provide basemap elevation coverage of the world in a raster tile format.
+
+![Contents of an example terrain tile](images/elevation-tile-example.png)
```
Comment (nvkelso) [inline docs/index.md]: Make a new Walkabout image!
